drivers/disk/sdmmc_stm32.c

```diff
@@ -151,6 +151,12 @@ static int stm32_sdmmc_clock_enable(struct stm32_sdmmc_priv *priv)
 	LL_RCC_SetSDMMCClockSource(LL_RCC_SDMMC1_CLKSOURCE_PLLSAI1);
 #endif
 
+#if defined(CONFIG_SOC_SERIES_STM32U5X)
+	/* By default the SDMMC clock source is set to 0 --> 48MHz, must be enabled */
+	LL_RCC_HSI48_Enable();
+	while (!LL_RCC_HSI48_IsReady()) {
+	}
+#endif /* CONFIG_SOC_SERIES_STM32U5X */
```

Comment:
This should be done using DT configuration.

Reply:
Ack waiting the HSI48 being available from device tree.

Reply:
see https://github.com/zephyrproject-rtos/zephyr/issues/49870 for moving the HSI48 to device tree